src/pages/index.js

```diff
@@ -1,35 +1,39 @@
 import React from "react"
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
-import { faChevronDown } from "@fortawesome/free-solid-svg-icons"
+import * as styles from "../utils/styles"
 import styled from "styled-components"
 
 import Link from "../components/Link"
 import PageMetadata from "../components/PageMetadata"
 import {
   PageBody,
-  Section,
+  // Section,
   ButtonLink,
+  Center,
   H2,
 } from "../components/SharedStyledComponents"
 import horzLogo from "../images/fellowship-program-logo.svg"
+import fellowsMap from "../images/fellows-map.png"
```

Comment:
I'd prefer if we used Gatsby Image for these images. Gatsby Image does a bunch of image optimization out of the both which improves performance.

Reply:
Totally agree! I also wanted to update Gatsby to v3, and iirc there's a different way to handle optimized images there - didn't want to start us out on a deprecated path. I'll get that done asap, just wanted to get this out for review by team quickly.

Reply:
Sounds good. FYI although it Gatsyb Image does change a bit, you can upgrade to v3 without changing anything re: Gatsby Image, e.g. see @wackerow's update on the ESP site: https://github.com/ethereum/esp-website/pull/84